### ISSUE TITLE
Dataset demo

### ISSUE THREAD

**Issue by yangzhikai** [VincentXWD/CreateGirlsMoe#4]
Could you put some image and conditional code as demo. The dataset is too big to download in a moment.

**Comment by VincentXWD** [comment]
1. Do you mean some generated images from the model? If so, check the homepage of this project. Otherwise please explain it in more detail.

2. I'm not sure what's the meaning of conditional code. Did you mean a script that use the trained model to generate batches of avatars? If so, please check [this file](https://github.com/VincentXWD/CreateGirlsMoe/blob/master/src/model/generate.py).

I have to admit that the dataset is large and it's hard for Chinese mainland users to download. If you meet troubles downloading the dataset according to the Google Drive, you can follow the steps mentioned in this [chapter](https://github.com/VincentXWD/CreateGirlsMoe#data-preparing) to construct your own dataset. It won't take you a few hours.

 Hope it helps.

**Comment by yangzhikai** [comment]
I mean the image and its tag for training. Because I want to use this project to generate other types of pictures rather cartoon.

**Comment by VincentXWD** [comment]
I think you'd better read the document carefully or glance the face detection of the paper. If you'd like to generate images in other type, please prepare dataset(both images and tags) on your own.

**Comment by yangzhikai** [comment]
I have similar images and labels, but I am not sure about the form of the input. And it need to be modified into the form of your project.

**Comment by VincentXWD** [comment]
For tags, convert the tag into one-hot vector; for images, resize the image and save the byte data in a list.
Please check [this](https://github.com/VincentXWD/CreateGirlsMoe/blob/master/src/model/data_preprocess.py#L48) part.
The list consists of many pairs of [tag one-hot, image]. Use this script to dump the whole data into a .dat file.

**Comment by yangzhikai** [comment]
Thanks, I would know something。

**Comment by VincentXWD** [comment]
The dataloader reads information from pickled dataset since the dataset is not so big. Dump it to a file can reduce some time consuming in IO stage. You can refactor the dataloader part especially [this](https://github.com/VincentXWD/CreateGirlsMoe/blob/master/src/model/data_loader.py#L11) line to fit the size of your dataset. Actually if you'd like to use your own dataset, the only one thing you have to do is to rewrite the dataloader.
